src/controller/tests/data_model/TestRead.cpp

```diff
@@ -4656,7 +4656,7 @@ System::Clock::Timeout TestReadInteraction::ComputeSubscriptionTimeout(System::C
 }
 
 // clang-format off
-const nlTest sTests[] =
+const nlTest sTestsOld[] =
```

Comment:
What is sTestsOld and where is it used?

Reply:
Used nowhere, just a trick to remove some tests for debugging. Reverted.

Reply:
A trick to remove some test for debugging. Reverted.